pandas/io/parquet.py

```diff
@@ -35,19 +36,15 @@ def get_engine(engine):
             "support"
         )
 
-    if engine not in ["pyarrow", "fastparquet"]:
-        raise ValueError("engine must be one of 'pyarrow', 'fastparquet'")
-
     if engine == "pyarrow":
         return PyArrowImpl()
     elif engine == "fastparquet":
         return FastParquetImpl()
 
+    raise ValueError("engine must be one of 'pyarrow', 'fastparquet'")
 
-class BaseImpl:
-
-    api = None  # module
```

Comment:
this is just unused?

Reply:
it doesn't appear to be necessary to be declared here. not used by the BaseImpl methods so not technically a mandatory property.

mypy errors were
```
pandas\io\parquet.py:101: error: "None" has no attribute "Table"
pandas\io\parquet.py:103: error: "None" has no attribute "parquet"
pandas\io\parquet.py:112: error: "None" has no attribute "parquet"
```